index2.html

```diff
@@ -68,6 +67,7 @@ <h1 class="work-2">Works</h1>
           <h1 class="project-name1">Multi-Post Stories</h1>
           <h1 class="project-name-two">Gain+Glory</h1>
```

Comment:
- [ ] Good job for addressing the previous reviewer's suggestions using `section` tag wrap each part of your site, cover, projects, about, etc.

- [Optional ] kindly, I suggest to follow the previous reviewer's advice for accessibility , it is not recommended to use more than 1 H1 tag.Please fix it